src/main/java/net/pms/util/ImdbUtil.java

```diff
@@ -99,8 +99,9 @@ public boolean accept(Path entry) throws IOException {
 						if (isBlank(entryName)) {
 							continue;
 						}
-						double score = StringUtils.getJaroWinklerDistance(nfoFileName, entryName);
-						if (score >= 0.85) {
+						double score = new JaroWinklerDistance().apply(nfoFileName, entryName);
+						if (score <= 0.85) { // threshold to add candidate to map. XXX it should be adjusted.
+											// The value 0.5 should be enough to insert relevant candidate.
```

Comment:
Can you change the value now instead of adding that comment?

Reply:
@SubJunk do you think that value 0.5 is OK or it could be less?

Reply:
@valib I don't know, it would probably need some testing with real files to get an idea of that. I don't know if there was much testing for our implementation of Jaro-Winkler. I tested it with dozens of files for the validation of prettifying, but not for choosing subtitles or NFO parsing.
I'm happy for you to change it to what you think is best.

Reply:
I wrote the above code, so yes it was quite a lot of testing and tweaking to try to find sane values. That said, you merged it before I finished it, so I'm not saying that those values are "ideal", just that I spent a lot of time trying to find a system that would be reasonably good at making the correct decisions.

Reply:
@valib the rest of the changes look good, are you happy with this change now or are you doing testing on it?